docker/Dockerfile-prow-env

```diff
@@ -0,0 +1,44 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM debian:bookworm
+
+LABEL maintainer="cloud-esf-dev@google.com"
+
+# add env we can debug with the image name:tag
+ARG IMAGE_ARG
+ENV IMAGE=${IMAGE_ARG}
+
+
+RUN apt-get update -y
+RUN apt-get -y install \
+    wget make cmake python3 python3-pip pkg-config coreutils \
+    zlib1g-dev curl libtool automake zip time rsync ninja-build \
+    git bash-completion jq default-jdk python3-distutils libicu-dev libbrotli-dev
+
+# # install Bazelisk
+RUN wget -O /usr/local/bin/bazelisk https://github.com/bazelbuild/bazelisk/releases/download/v1.15.0/bazelisk-linux-amd64 && \
+    chmod +x /usr/local/bin/bazelisk && \
+    cp /usr/local/bin/bazelisk /usr/local/bin/bazel
```

Comment:
nit: IMO it's better to not symlink and just change our CI script to use the `bazelisk` command instead of `bazel`. But not a big deal for CI image. It makes more of a difference for your local workstation